packages/at_login_flutter/pubspec.yaml

```diff
@@ -8,29 +8,28 @@ environment:
   flutter: ">=1.20.0"
 
 dependencies:
+  at_client: ^3.0.64
+  at_client_mobile: ^3.2.12
+  at_commons: ^3.0.55
+  at_server_status: ^1.0.3
+  at_utils: ^3.0.15
+  basic_utils: ^5.6.1
+  file_picker: ^5.5.0
   flutter:
     sdk: flutter
-  provider: ^5.0.0
-  permission_handler: ^8.1.2
-  at_client: ^3.0.59
-  at_client_mobile: ^3.2.9
-  at_commons: ^3.0.0
-  at_server_status: ^1.0.3
-  at_utils: ^3.0.0
-  file_picker: ^4.0.0
-  flutter_local_notifications: ^5.0.0+1
+  flutter_local_notifications: ^15.1.1
```

Comment:
flutter_local_notifications ^5.0.0+1 -> ^15.1.1 
Yes, you're reading that right... 10 major versions...